{"name": "test_login_2_success_login", "status": "failed", "statusDetails": {"message": "AssertionError: assert 'https://www....er/login.html' == 'https://www.k...in/index.html'\n  - https://www.ketangpai.com/Home/User/login.html\n  ?                           ^^^^^^^^^^^^^\n  + https://www.ketangpai.com/Main/index.html\n  ?                           ^^  ++++++", "trace": "self = <TestCases.test_loginpai.TestLogin object at 0x0000017B70D2E128>\nopen_url = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"dce870436c33588cfb5e27a1c9603c7e\")>\n\n    def test_login_2_success_login(self, open_url):\n            # 步骤 - 登陆操作 - 登陆页面 - 18684720553、python\n            LoginPage(open_url).login(ld.success_data[\"user\"], ld.success_data[\"passwd\"])  # 测试对象+测试数据\n            # 断言 - 页面是否存在   我的帐户   元素   元素定位+元素操作\n            # self.assertTrue(IndexPage(self.driver).check_nick_name_exists())  # 测试对象+测试数据\n            # url跳转\n>           assert open_url.current_url==ld.success_data[\"check\"] # 测试对象+测试数据\nE           AssertionError: assert 'https://www....er/login.html' == 'https://www.k...in/index.html'\nE             - https://www.ketangpai.com/Home/User/login.html\nE             ?                           ^^^^^^^^^^^^^\nE             + https://www.ketangpai.com/Main/index.html\nE             ?                           ^^  ++++++\n\nTestCases\\test_loginpai.py:39: AssertionError"}, "start": 1562259120782, "stop": 1562259121399, "uuid": "12d9005f-f59b-4bc4-a608-a23950bbd3e2", "historyId": "ec61490cd352bd98b2c9d9e9683c793b", "fullName": "TestCases.test_loginpai.TestLogin#test_login_2_success_login", "labels": [{"name": "parentSuite", "value": "TestCases"}, {"name": "suite", "value": "test_loginpai"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "DESKTOP-F99FSHQ"}, {"name": "thread", "value": "16424-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "TestCases.test_loginpai"}]}